docs/pbc/all/pbc.md

```diff
@@ -63,11 +63,12 @@ This section is in beta because not all the PBCs are covered. Marketplace functi
 
 <div class="width-100">
 
-| NAME | CATEGORY |
-| --- | --- |
-| [Algolia](/docs/pbc/all/search/{{site.version}}/base-shop/third-party-integrations/algolia/algolia.html) | Search |
-| [Bazaarvoice](/docs/pbc/all/ratings-reviews/{{site.version}}/third-party-integrations/bazaarvoice.html) | Ratings and Reviews |
+| NAME                                                                                                                                                                 | CATEGORY |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------| --- |
+| [Algolia](/docs/pbc/all/search/{{site.version}}/base-shop/third-party-integrations/algolia/algolia.html)                                                             | Search |
+| [Bazaarvoice](/docs/pbc/all/ratings-reviews/{{site.version}}/third-party-integrations/bazaarvoice.html)                                                              | Ratings and Reviews |
 | [Payone](/docs/pbc/all/payment-service-provider/{{site.version}}/base-shop/third-party-integrations/payone/app-composition-platform-integration/payone-acp-app.html) | PSP (Payment Service Provider) |
-| [Usercentrics](/docs/pbc/all/usercentrics/usercentrics.html) | CMP (Consent Management Platform) |
+| [Stripe](/docs/pbc/all/payment-service-provider/{{site.version}}/base-shop/third-party-integrations/stripe/stripe.html) | PSP (Payment Service Provider) |
```

Comment:
```suggestion
| [Stripe](/docs/pbc/all/payment-service-provider/{{site.version}}/base-shop/third-party-integrations/stripe/stripe.html)                                                              | PSP (Payment Service Provider) |
```

add more spaces to have table columns aligned with other rows

Reply:
I don't think that we have to have the same spaces. Should work out of the box by the separator.